src/conventions/h-group/clue-finder/determine-clue.js

```diff
@@ -33,6 +33,11 @@ export function evaluate_clue(game, action, clue, target, target_card, bad_touch
 	logger.highlight('green', `------- ENTERING HYPO ${logClue(clue)} --------`);
 
 	const hypo_game = game.simulate_clue(action, { enableLogs: true });
+	// This is emulating the needed side effects of handle_action for a clue action.
+	// It might be simpler to call handle_action on the hypo_game.
+	hypo_game.last_actions = game.last_actions.slice();
```

Comment:
Is it necessary to slice `last_actions`? `simulate_clue()` already creates a copy of the game, so any changes to `hypo_game` shouldn't affect the original game.